### DISCUSSION TITLE
Ionic 5 Capacitor - Ignore Android accessibility font size

### DISCUSSION THREAD

**Discussion by nachshonfertel** [ionic-team/capacitor#3706] [Q&A]
I'm using Ionic 5 with Capacitor, however, whenever an Android user has their accessibility settings font-size set to large, many parts of the app get deformed. I'm trying to ignore the phone settings and keep the font-size regular.

I've tried using the Ionic Mobile Accessibility plugin as answered here, but that does not seem to work with Capacitor.
https://stackoverflow.com/questions/54450312/ionic-3-application-how-to-make-font-size-independent-of-native-settings

![image](https://user-images.githubusercontent.com/18723408/96599072-87b77c00-12bd-11eb-8346-5c4177ea06cc.png)

**Comment by jacksonwc2** [comment]
Hello, I had the same problem, but I managed to solve it using the following:

Executei a instalação do plugin nativo, conforme orientação da documentação para capacitor:
https://ionicframework.com/docs/native/mobile-accessibility#installation
Installation > Capacitor.

`npm install phonegap-plugin-mobile-accessibility
npm install @ionic-native/mobile-accessibility
ionic cap sync`

After that, I added a provider in App.module.ts:

`...
 import { MobileAccessibility } from '@ionic-native/mobile-accessibility/ngx';
... 
@NgModule({
...
  providers: [
...
    MobileAccessibility,
...
  ],
 ...
})
export class AppModule {}
`


Finally, the following code snippet was added when launching app.component.ts:

`this.mobileAccessibility.setTextZoom(100);
    this.mobileAccessibility.updateTextZoom();
    this.mobileAccessibility.usePreferredTextZoom(false);`

**Reply by shravanvis** [reply]
any cordova plugin for this
im using cordova in ionic

**Reply by kensodemann** [reply]
The plugin that @jacksonwc2 is using (https://github.com/phonegap/phonegap-mobile-accessibility) _is_ a Cordova plugin.

The @ionic-native wrappers are deprecated and you [should be using these](https://github.com/danielsogl/awesome-cordova-plugins) instead. That particular plugin didn't make the cut, though, because it hasn't been maintained at all in five years. As such, you may need to either just use the plugin directly or better yet, create your own Angular service to abstract the calls to it.

This also means if the plugin has a bug or is missing functionality, you _may_ need to fork it and fix it.

All of that said, I would also say a way better way of dealing with this is to rework your app to actually be accessible to people who need the larger font.

**Reply by shravanvis** [reply]
[ng] Error: node_modules/@ionic-native/google-plus/ngx/index.d.ts:1:35 - error TS2307: Cannot find module '@ionic-native/core' or its corresponding type declarations.
[ng] 1 import { IonicNativePlugin } from '@ionic-native/core';
[ng]                                     ~~~~~~~~~~~~~~~~~~~~



it gave me errors in other native when I'm installing it

**Comment by rabbitom** [comment]
Use [@capacitor/text-zoom](https://ionicframework.com/docs/native/text-zoom) instead
